pre-docs/bolt_project_directory.md

```diff
@@ -0,0 +1,115 @@
+# Bolt Project Directory
+
+Bolt runs in the context of a bolt project directory or a `Boltdir`. This
+directory contains all of the configuration, code and data loaded by bolt. This
+makes it easy to share bolt code by comitting the project directory to git and
+have multiple contexts for bolt checked out into different directories for
```

Comment:
This structure makes it easy to share Bolt code by committing the project directory to Git. You can then check out Bolt code into different directories in order to various applications.